### PR TITLE
fix typo in vgg.md

### PR DESCRIPTION
**Pull request by WilliamWuLH**
fix typo in vgg.md

### CODE REVIEW

**Review by WilliamWuLH — COMMENTED** [review]
fix typo in vgg.md

**Comment by mli** [comment]
Job d2l-zh/PR-745/2 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-745/

File: chapter_convolutional-modern/vgg.md [hunk 1]
```diff
@@ -12,8 +12,8 @@
 
 经典卷积神经网络的基本组成部分是下面的这个序列：
 1. 带填充以保持分辨率的卷积层；
-1. 非线性激活函数，如ReLU；
-1. 池化层，如最大池化层。
+2. 非线性激活函数，如ReLU；
+3. 池化层，如最大池化层。
```
Comment (goldmermaid) [inline chapter_convolutional-modern/vgg.md:16]: Please revert back to the original. `.md` file will auto-increase the number sequence when it's rendering.

**Comment by mli** [comment]
Job d2l-zh/PR-745/3 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-745/

**Comment by mli** [comment]
Job d2l-zh/PR-745/4 is complete. 
Check the results at http://preview.d2l.ai/d2l-zh/PR-745/

**Review by mehdits90 — APPROVED** [review]
-

**Pull request merged**